doc/source/user/installation.rst

```diff
@@ -76,9 +76,56 @@ Use ``pip`` to remove FuseSoC from your system.
 Installation under Windows
 ==========================
 
-.. todo::
+FuseSoC is provided as ``fusesoc`` Python package and installed through pip, the Python package manager.
+Python is not installed by default on Windows, so doing that is the first step.
+Install the latest version of Python, either from `python.org <https://www.python.org/downloads/>`_ or from the Windows store.
+
+Open up a shell (cmd, powershell, gitbash etc.) and run:
+
+.. code-block:: shell-session
+
+   $ python --version
+
+If the output is something along the lines of ``Python 3.10.7``, Python has been successfully installed.
+
+
+Installation for the current user
+---------------------------------
+
+To install the current stable version of FuseSoC for the current user, open a shell and run the following command.
+If an older version of FuseSoC is found, this version is upgraded to the latest stable release.
+
+.. code-block:: shell-session
+
+   $ pip3 install --upgrade --user fusesoc
+
+Check that the installation worked by running
+
+.. command-output:: fusesoc --version
 
-    Add Windows installation instructions.
+
+If this command works FuseSoC is installed properly and ready to be used.
+
+If the shell reports an error about the command not being found verify that the ``Scripts`` directory (example: ``C:/Users/youruser/AppData/Roaming/Python/Python310/Scripts``) folder is added to the search path (``PATH``).
+
+
+System-wide installation
+------------------------
+
+FuseSoC can be installed for all users on a system.
+
+.. code-block:: shell-session
+
+   $ pip3 install --upgrade fusesoc
```

Comment:
As you mentioned, both installations are "per user". So simply remove the user installation instructions above, keeping only the "system ones" (and make it the one and only instructions).

Reply:
Will fix.